docs/metrics/ingest.md

```diff
@@ -55,3 +55,17 @@ The timestamp provided should be:
 - We don’t guarantee correct behavior if metrics are provided with an earlier timestamp after this API is called.
 
 Statsig processes metrics as a full day in the PST timezone; we will wait until a full day is marked as ready before processing that day.
+
+## Debugging Precomputed Metrics 
+
+All precomputed metrics generate Metric Detail View pages. However, these Detail View pages take a few hours to generate post-ingestion. The fastest way to start seeing and debugging your precomputed metrics is via the **Metrics Logstream** on the **Metrics Catalog** tab within **Metrics**. 
+
+![Screen Shot 2022-07-13 at 4 22 46 PM](https://user-images.githubusercontent.com/101903926/178854882-730ef0d5-8eb2-4344-88ab-33111301e712.png)
+
```

Comment:
Nit: this is in the first step of ingestion. If they don’t show up, there is a connection or schema issue